react-components/toolbar-filter/step6.md

```diff
@@ -1,95 +1,13 @@
-One particular type of Data toolbar item is called the `DataToolbarFilter`. When used, selected filters are displayed in chip groups below the toolbar. You may consider placing any `DataToolbarFilter` inside of a toggle group to make your toolbar responsive.
+Managing the relationship of a toolbar with its filters to a table or other component is the responsibility of the consumer. 
 
-Using a `DataToolbarFilter` requires three properties. First is a managed array of selected filters called `chips` as strings or ReactNodes. It also requires an `onDelete` callback function to be executed whenever the user deletes a selected filter chip. Lastly, it requires a `categoryName` which will be used to label the chip group.
-
-## Task
-1) Define the required `onDelete` function to be passed to the DataToolbarFilters. Add it to the ToolbarDemoApp class.
-
-<pre class="file" data-target="clipboard">
-  onDelete = (type = &quot;&quot;, id = &quot;&quot;) =&gt; {
-    if (type) {
-      this.setState((prevState) =&gt; {
-        prevState.filters[type.toLowerCase()] = prevState.filters[type.toLowerCase()].filter(s =&gt; s !== id);
-        return {
-          filters: prevState.filters,
-        }
-      });
-    } else {
-      this.setState({
-        filters: {
-          name: [],
-          risk: [],
-          status: [],
-        }
-      })
-    }
-  };
-</pre>
-
-2) Locate the following three `DataToolbarItems` inside the `DataToolbarToggleGroup`:
-<pre class="file">
-  &lt;DataToolbarItem variant=&quot;search-filter&quot;&gt;
-    &lt;Input value=&quot;&quot; id=&quot;step2Input&quot; ariaLabel=&quot;Step 2 input&quot; /&gt;
-  &lt;/DataToolbarItem&gt;
-  &lt;DataToolbarGroup variant=&quot;filter-group&quot;&gt;
-    &lt;DataToolbarItem&gt;
-      &lt;CheckboxSelect
-        onSelect={this.onSelect}
-        type=&quot;Status&quot;
-        selections={filters.status}
-        options={statusOptions}
-      /&gt;
-    &lt;/DataToolbarItem&gt;
-    &lt;DataToolbarItem&gt;
-      &lt;CheckboxSelect
-        onSelect={this.onSelect}
-        type=&quot;Risk&quot;
-        selections={filters.risk}
-        options={riskOptions}
-      /&gt;
-    &lt;/DataToolbarItem&gt;
-  &lt;/DataToolbarGroup&gt;
-</pre>
-
-3) Replace each of the three `DataToolbarItems` inside the `DataToolbarToggleGroup` with a `DataToolbarFilter`. Pass the appropriate state managed filters array as the `chips` property, an appropriate label as the `categoryName` property, and the onDelete function as the `deleteChip` property. 
+1) Add the `DemoTable` component below the `DataToolbar` and pass it the status and risk filters as props.
```

Comment:
Make this bold. ALso theres a lot going on in this step, so separate it out by first having the overall action of the step, and then putting the other information in its own sentence